qhub/render/__init__.py

```diff
@@ -92,16 +88,11 @@ def deep_merge(d1, d2):
         return d1
 
 
-def render_default_template(output_directory, config_filename=None, force=False):
+def render_default_template(output_directory, config_filename, force=False):
```

Comment:
if there is only one function for rendering template, then probably worth naming it as just `render_template`?

Reply:
Sure, have done now. Shorter name is better!